tests/test_actor.py

```diff
@@ -137,10 +141,36 @@ def test_independent_cleanup_continues_through_exceptions():
 
 
 def test_forgets_abilities_when_exiting():
-    mocked_ability = mock.Mock()
+    mocked_ability = mock.create_autospec(Ability, instance=True)
     actor = Actor.named("Tester").who_can(mocked_ability)
 
-    actor.exit()
+    actor.exit_stage_left()
 
     mocked_ability.forget.assert_called_once()
     assert len(actor.abilities) == 0
+
+
+def test_exit_stage_right():
+    mocked_ability = mock.create_autospec(Ability, instance=True)
+    actor = Actor.named("Tester").who_can(mocked_ability)
+
+    actor.exit_stage_right()
+
+    mocked_ability.forget.assert_called_once()
+
+
+def test_exit_through_vomitorium():
+    mocked_ability = mock.create_autospec(Ability, instance=True)
+    actor = Actor.named("Tester").who_can(mocked_ability)
+
+    actor.exit_through_vomitorium()
+
+    mocked_ability.forget.assert_called_once()
```

Comment:
I think we could probably use `pytest.mark.parametrize` to test all these versions of `exit`.